### PR TITLE
fix(simulator) : fixes the username display in simulator

### LINKED ISSUE #4519: 🐞 Bug The username is partially obscured when the simulator screen is minimized.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The username of the user in the navbar is cut in half when the screen is minimized in the simulator.

### Steps To Reproduce

1. Signin into the circuitverse (https://circuitverse.org/users/sign_in)
2. Go to https://circuitverse.org/simulator
3. Check username of yours 

### Expected Behavior

The navbar should display the entire username of the user in the simulator.

### Device Information [optional]

```markdown
- OS: Windows 11
- Browser: Brave
- version:
```

### Additional context

_No response_

### Are you working on this issue?

Yes

### PR DESCRIPTION
**Pull request by aryanndwi123**
Fixes #4519 
#### Describe the changes you have made in this PR -
Username of the person was not visible when simulator screen minimized

### Screenshots of the changes (If any) -

Before changes:

After changes:

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by aryanndwi123** [comment]
Hi @Prerna-0202 @Arnabdaz ,
Can you review this pr and tell me the required changes.

**Comment by Arnabdaz** [comment]
please link the issue for which this fix is.

**Comment by aryanndwi123** [comment]
> please link the issue for which this fix is.
Sure @Arnabdaz 
I have linked it..!

File: simulator/src/css/main.stylesheet.css [hunk 1]
```diff
@@ -177,7 +177,7 @@ input[type="text"]:focus {
     height: 1.2rem;
     overflow: hidden;
     text-overflow: ellipsis;
-    margin-right: 22rem;
+
```
Comment (tanmoysrt) [inline simulator/src/css/main.stylesheet.css:180]: Instead of removing use
`margin-right: auto`

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

File: simulator/src/css/main.stylesheet.css [hunk 2]
```diff
@@ -177,7 +177,8 @@ input[type="text"]:focus {
     height: 1.2rem;
     overflow: hidden;
     text-overflow: ellipsis;
-    margin-right: 22rem;
+    margin-right: auto;
+
```
Comment (tanmoysrt) [inline simulator/src/css/main.stylesheet.css:181]:
```suggestion
```

**Review by tanmoysrt — APPROVED** [review]
Looks good :+1:

**Pull request merged**